packages/theme/src/themes/bruttalTheme.ts

```diff
@@ -0,0 +1,96 @@
+import { Theme } from 'theme-ui';
+
+export const bruttalTheme: Theme = {
```

Comment:
I think it'd be better to use PascalCase. `BruttalTheme`, as you did on Storybook `import { bruttalTheme as Bruttal } from '@ttoss/theme';`